base/Audio.ts

```diff
@@ -7,6 +7,15 @@ export class Audio {
         this.game = game;
     }
 
+    /**
+     * Changes the game volume.
+     * @param {number} delta - Delta to apply to the volume
+     * @return {number} Final game volume
+     */
+    alter_volume(delta: number) {
+        return (this.game.sound.volume = Math.max(Math.min(this.game.sound.volume + delta, 1), 0));
```

Comment:
Use lodash in order to have this more readable: `_.clamp(this.game.sound.volume + delta, 0, 1)`